test/t_web/channels/feed_channel_test.exs

```diff
@@ -72,17 +72,18 @@ defmodule TWeb.FeedChannelTest do
                  "id" => m3.id,
                  "profile" => %{name: "mate-3", story: [], user_id: p3.id, gender: "M"},
                  "timeslot" => %{"selected_slot" => s2},
-                 "audio_only" => false
+                 "audio_only" => false,
+                 "expiration_date" => ~U[2021-10-07 12:16:07Z]
```

Comment:
@akhrail1996 since the matches didn't have started calls and reported meetings, `expiration_date` should be returned

Reply:
@ruslandoga yes, I don't remember why exactly it wasn't rendered here (though it didn't affect real usage) - but it should be